src/tx-format/transaction.md

```diff
@@ -51,29 +49,31 @@ enum ReceiptType : uint8 {
     TransferOut = 8,
     ScriptResult = 9,
     MessageOut = 10,
-    Mint = 11
-    Burn = 12
+    Mint = 11,
+    Burn = 12,
 }
 ```
 
-| name               | type                        | description                                          |
-|--------------------|-----------------------------|------------------------------------------------------|
-| `gasPrice`         | `uint64`                    | Gas price for transaction.                           |
-| `gasLimit`         | `uint64`                    | Gas limit for transaction (including predicate gas). |
-| `maturity`         | `uint32`                    | Block until which tx cannot be included.             |
-| `scriptLength`     | `uint16`                    | Script length, in instructions.                      |
-| `scriptDataLength` | `uint16`                    | Length of script input data, in bytes.               |
-| `inputsCount`      | `uint8`                     | Number of inputs.                                    |
-| `outputsCount`     | `uint8`                     | Number of outputs.                                   |
-| `witnessesCount`   | `uint8`                     | Number of witnesses.                                 |
-| `receiptsRoot`     | `byte[32]`                  | Merkle root of receipts.                             |
-| `script`           | `byte[]`                    | Script to execute.                                   |
-| `scriptData`       | `byte[]`                    | Script input data (parameters).                      |
-| `inputs`           | [Input](./input.md)`[]`     | List of inputs.                                      |
-| `outputs`          | [Output](./output.md)`[]`   | List of outputs.                                     |
-| `witnesses`        | [Witness](./witness.md)`[]` | List of witnesses.                                   |
+| name               | type                        | description                            |
+|--------------------|-----------------------------|----------------------------------------|
+| `scriptLength`     | `uint16`                    | Script length, in instructions.        |
+| `scriptDataLength` | `uint16`                    | Length of script input data, in bytes. |
+| `policyTypes`      | `uint32`                    | Bitfield of used policy types.         |
```

Comment:
I think it's okay to use `u64` here since it takes 8 bytes in canonical format.

```suggestion
| `policyTypes`      | `uint64`                    | Bitfield of used policy types.         |
```

Reply:
True but it could save data outside of the VM for things like DA